src/components/beregningOgSimulering/beregning/Beregning.tsx

```diff
@@ -246,7 +250,9 @@ const Beregning = (props: VilkårsvurderingBaseProps) => {
                                     null
                                 }
                                 feltnavn="fradrag"
-                                fradrag={formik.values.fradrag}
+                                fradrag={formik.values.fradrag.filter(
+                                    (fradrag) => fradrag.type !== Fradragstype.AvkortingUtenlandsopphold
```

Comment:
Føles det riktig at AvkortingUtenlandsopphold hører hjemme på denne typen? Kanskje vi bør splitte opp i forskjellige fradragstyper basert på input/beregnet?

Reply:
Man kunne splittet de opp ja. I to forskjellige fradragstyper. Jeg fulgte egentlig bare den tankegangen som allerede var implementert